x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list_table.tsx

```diff
@@ -143,7 +143,8 @@ export interface RulesListTableProps {
     onLoading: (isLoading: boolean) => void
   ) => React.ReactNode;
   renderRuleError?: (rule: RuleTableItem) => React.ReactNode;
-  visibleColumns?: RulesListVisibleColumns[];
+  visibleColumns?: string[];
+  columns?: RulesListColumns[];
```

Comment:
Why did we add this `columns` prop here?
What the difference between `visibleColumns` and `columns`?

Reply:
we added because we would like user of this table to add their own columns if they want to

Reply:
I ended up removing this part of the code, can be recovered here https://github.com/elastic/kibana/pull/147014/commits/7d240fa2bbc377cec5491e03537ba6b0d8e478b8